### DISCUSSION TITLE
Hide "TitleBarMinMaxCloseContainer"

### DISCUSSION THREAD

**Discussion by Petrarca181** [microsoft/microsoft-ui-xaml#8425] [Q&A] [answered]
Hello, as currently not possible to hide  maximize/minimize  button I was using this:

`Style TargetType="ContentControl" x:Key="WindowChromeStyle">`
               `<Setter Property="Template">`
                   `<Setter.Value>`
                      ` <ControlTemplate TargetType="ContentControl">`
                          `<Grid x:Name="LayoutRoot">`
                              `<Grid x:Name="TitleBarMinMaxCloseContainer" Visibility="Collapsed">`
                                  `<Grid x:Name="MinMaxCloseContainer"/>`
                                `</Grid>`
                               `<ContentPresenter x:Name="ClientAreaPresenter"`
                           ` Content="{TemplateBinding Content}"`
                         ` ContentTemplate="{TemplateBinding ContentTemplate}"`
                           ` ContentTransitions="{TemplateBinding ContentTransitions}"`
                          ` Foreground="{TemplateBinding Foreground}"/>`
                           `</Grid>`
                   `</ControlTemplate>`
                  `</Setter.Value>`
             `</Setter>`
          `</Style>`
But now I'm using WinUIEX  and that style crashes  my app.   So I tried this:
   WindowManager.Get(this).IsMinimizable = false;
   WindowManager.Get(this).IsMaximizable = false;

still nothing. Any Ideas?  It's possible to navigate through VisualTreeHelper  in code behind to the  ContentControl?

**Comment by DarranRowe** [comment, marked as answer]
It would be nice to know precisely what you are doing here.

![Screenshot 2023-04-21 224551](https://user-images.githubusercontent.com/52577874/233739521-008300a9-d02e-45f8-8a46-bf41a5b10698.png)

The standard title bar removes these buttons without issue.

![Screenshot 2023-04-21 224803](https://user-images.githubusercontent.com/52577874/233739864-ea33e9cb-3799-4620-a103-64a8a252eae8.png)

Microsoft.UI.Windowing.AppWindowTitleBar.ExtendsContentIntoTitleBar removes these buttons without issue.

![Screenshot 2023-04-21 224945](https://user-images.githubusercontent.com/52577874/233740035-8acb8066-0fe9-4dbc-95fa-a3aadc436128.png)

It is only Microsoft.UI.Xaml.Window.ExtendsContentIntoTitleBar that ignores the window style.

So it is possible, the issue is that one of the methods to extend the content into the title bar does a poor job of mirroring the style set on the window, and the use of AppWindowTitleBar is the recommended method, assuming there are no bugs.

**Comment by Petrarca181** [comment]
I cleared bin and obj folder  now after rebuilding the project  WindowManager.Get(this).IsMinimizable = false;
WindowManager.Get(this).IsMaximizable = false;   hide those buttons.  Idk why and how.
